x-pack/legacy/plugins/infra/public/components/logging/log_analysis_results/analyze_in_ml_button.tsx

```diff
@@ -8,10 +8,11 @@ import { EuiButton } from '@elastic/eui';
 import { FormattedMessage } from '@kbn/i18n/react';
 import React from 'react';
 import { encode } from 'rison-node';
-import { QueryString } from 'ui/utils/query_string';
 import url from 'url';
+import { stringify } from 'query-string';
 import { useKibana } from '../../../../../../../../src/plugins/kibana_react/public';
 import { TimeRange } from '../../../../common/http_api/shared/time_range';
+import { url as urlUtils } from '../../../../../../../../src/plugins/kibana_utils/public';
```

Comment:
Someday soon I hope we figure out a way to cleanly avoid this because it's awful to look at and horribly error-prone :(

Reply:
This is the relevant PR to check out for that if you aren't already following it https://github.com/elastic/kibana/pull/52995  :)